unix_integration/src/tasks_daemon.rs

```diff
@@ -135,16 +169,48 @@ fn create_home_directory(
                         .strip_prefix(skel_dir)
                         .map_err(|e| e.to_string())?,
                 );
+
+                #[cfg(all(target_family = "unix", feature = "selinux"))]
+                {
+                    // Look up the correct SELinux context of this object
+                    let sel_lookup_path = Path::new(&sel_lookup_path_raw).join(
+                        entry
+                            .path()
+                            .strip_prefix(skel_dir)
+                            .map_err(|e| e.to_string())?,
+                    );
+                    selinux_util::do_setfscreatecon_for_path(
+                        &sel_lookup_path.to_str().unwrap().to_string(),
+                        &labeler,
+                    )?;
+                }
+
                 if entry.path().is_dir() {
                     fs::create_dir_all(dest).map_err(|e| e.to_string())?;
                 } else {
                     fs::copy(entry.path(), dest).map_err(|e| e.to_string())?;
                 }
                 chown(dest, info.gid)?;
+
+                // Create equivalence rule in the SELinux policy
+                #[cfg(all(target_family = "unix", feature = "selinux"))]
+                if Command::new("semanage")
+                    .args(["fcontext", "-ae", &sel_lookup_path_raw, &hd_path_raw])
```

Comment:
I imagine that the reason for the command here is that the selinux crate doesn't support this?

Reply:
Correct -- I didn't see any way to do it with the crate, and in any case this is better from a policy perspective because it allows writing a rule to allow `kanidm_unixd_tasks` to execute the `semanage` binary instead of having all of the privileges needed to do the policy changes itself.